src/main/java/dev/ftb/mods/ftbic/block/entity/generator/NuclearReactorBlockEntity.java

```diff
@@ -51,6 +51,7 @@ public class NuclearReactorBlockEntity extends GeneratorBlockEntity {
 	public static final int[] OFFSET_Y = {-1, 1, 0, 0};
 
 	public static final SyncedDataKey<Double> ENERGY_OUTPUT = new SyncedDataKey<>("energy_output", 0D);
+	public static final double ENERGY_MULTIPLIER = FTBICConfig.MACHINES.NUCLEAR_GENERATOR_OUTPUT.get();
```

Comment:
Please don't store the config value as a static final in this clas, there is no need as it's just going to act as a supplier reference... :P